app/controllers/api/v01/items_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V01::ItemsController < Api::V01::GeneralController
+  include LogWrapper
+
+  before_filter :set_request_body
+  before_filter :validate_source_of_request
+
+  #Here Updating the available_copies, total copies in Teacher Set table.
+  def update_availability
+    begin
+      http_status = 200
+      error_code_and_message = validate_request
+      if error_code_and_message.any?
+        AdminMailer.failed_items_controller_api_request(@request_body, error_code_and_message, action_name).deliver
+        render_error(error_code_and_message)
+      end
```

Comment:
Do you need a 
`return if error_code_and_message.any?`
?

Reply:
I didnt see any use case with this  line of code. So i didnt added return if error_code_and_message.any?